docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
 # Control of documents
 
-Bioinformaticians work in a git environment. Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordination work among programmers collaboratively developing source code during software development. Software procedures and related documentation needs to be controlled, and according to ISO-15189, this can be in any form or type of medium. For software development, there are several reasons to not include all the software documentation within the medical laboratories chosen quality management system.
+## Why do Bioinformaticians need a different QMS
```

Comment:
Just a detail; either remove 'do' or add a question mark at the end

Reply:
or "Bioinformaticians can benefit from a different QMS"